<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QMainWindow" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnViewMyAccounts">
      <property name="text">
       <string>View My Accounts</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCardToCard">
      <property name="text">
       <string>Card to Card</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEditProfile">
      <property name="text">
       <string>Edit Profile</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>454</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
